solution/server/src/types/types.ts

```diff
@@ -0,0 +1,310 @@
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
How would you do to prevent engineers to forget pushing this file when schema changes?

Reply:
My suggestion would be to implement a GitHub Action (or any CI routine) to run the Codegen on every commit and evaluate if there are any diffs between the generated and the repository's current "types.ts". A rule must be added to accept commits and allow merging branches only when there are no diffs between this file and the one generated by the action.